src/tests/RnR_tests/RnR.test.jsx

```diff
@@ -27,22 +27,24 @@ describe('List and sub components exist', () => {
 });
 
 // Tests for List
-function testList() {
-  const props = {
-  };
-  const enzymeWrapper = shallow(<List />);
-  return {
-    props,
-    enzymeWrapper,
-  };
-}
+// Test no longer applicable
```

Comment:
Unless you're saving this code to reuse later it should probably just be deleted.

Reply:
same with the test lower down